host/src/channel_manager.rs

```diff
@@ -109,7 +109,7 @@ impl<'d, P: PacketPool> ChannelManager<'d, P> {
         }
     }
 
-    fn next_request_id(&self) -> u8 {
+    pub(crate) fn next_request_id(&self) -> u8 {
```

Comment:
I would prefer if this was crate internal to avoid it leaking into other places

Reply:
Fixed